node/sc/subbridge.go

```diff
@@ -520,14 +521,16 @@ func (pm *SubBridge) handle(p BridgePeer) error {
 
 			if pm.peers.Len() == 1 {
```

Comment:
Don't we need to reset bridge subscriptions if Len() != 1?

Reply:
This code means `'error occurs while reading from a peer'` and `'the peer is the last one'`.

Reply:
Fix the reset condition. PTAL.